config/sites/facilities.uiowa.edu/config_split.patch.core.entity_form_display.node.person.default.yml

```diff
@@ -4,6 +4,7 @@ adding:
       - field.field.node.person.field_person_birth_year
       - field.field.node.person.field_person_death_year
       - field.field.node.person.field_person_nationality
+      - workflows.workflow.editorial
```

Comment:
This change seems odd to me.

Reply:
I think this is a config export blip as it is captured in config/default/core.entity_form_display.node.person.default.yml

I don't think it breaks import if it isn't there if we want to remove it.

Reply:
I'd say we remove it because we won't remember why its there eventually.

Reply:
it is in a few config patches for ITS as well. so it has been sneaking in outside of this 10.2 update

Reply:
It's not a big deal either way. I'm okay if we don't do anything.

Reply:
>I'd say we remove it because we won't remember why its there eventually.

works for me. Removed from wellbeing, facilities and its. running syncs on those sites were successful locally.